app/src/main/java/com/github/sdp/tarjetakuna/ui/singlecard/SingleCardFragment.kt

```diff
@@ -168,4 +188,15 @@ class SingleCardFragment : Fragment() {
         super.onDestroyView()
         _binding = null
     }
+
+    private inner class ItemFragmentAdapter(fa: FragmentActivity) : FragmentStateAdapter(fa) {
+        override fun getItemCount(): Int = 2
```

Comment:
Consider defining this constant outside the function

Reply:
I changed that to `binding.singleCardTabLayout.tabCount`. Thanks for your remark.